python/tk_framework_adobe/rpc/proxy.py

```diff
@@ -232,7 +235,7 @@ def __iter__(self):
                     yield self[i]
                     i = i + 1
             except IndexError:
-                raise StopIteration
+                return
```

Comment:
It seems in Python3 you should return instead of raising a StopIteration error, as it actually just causes the program to error. `rerturn` appears to still work in Python 2.

Reply:
Totally. generators will raise a StopIteration error themselves in Python when trying to get get items past the end of the collection.